Minesweeper.py

```diff
@@ -99,6 +99,8 @@ def game_loop():
             if board[player_pos[1]][player_pos[0]] == 0:
                 board[player_pos[1]][player_pos[0]] = "F"
                 flags += 1
+            elif board[player_pos[1]][player_pos[0]] == "F":  # Error: Redundant condition, already checked in unflag
```

Comment:
The condition here seems redundant since it is already checked in an earlier if condition regarding flags.
```diff
-                elif board[player_pos[1]][player_pos[0]] == "F":  # Error: Redundant condition, already checked in unflag
+                # elif board[player_pos[1]][player_pos[0]] == "F":  # Redundant condition, already checked in unflag
```
---